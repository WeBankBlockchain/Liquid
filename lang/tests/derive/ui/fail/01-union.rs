use liquid::InOut;
use liquid_lang as liquid;

#[derive(InOut)]
#[repr(C)]
pub union MyUnion {
    f1: u32,
    f2: f32,
}

fn main() {}
