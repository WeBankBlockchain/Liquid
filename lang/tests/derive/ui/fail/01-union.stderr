error: unions are not supported
 --> $DIR/01-union.rs:6:5
  |
6 | pub union MyUnion {
  |     ^^^^^
